src/omnicore/tx.cpp

```diff
@@ -82,6 +82,16 @@ bool CMPTransaction::isOverrun(const char* p)
 
 // -------------------- PACKET PARSING -----------------------
 
+/** Temporarily disables packet debugging and then calls interpret_Transaction(). */
+bool CMPTransaction::reinterpret_Transaction()
+{
+    bool packetDebugState = msc_debug_parser;
+    msc_debug_parser = 0;
```

Comment:
Well, I kind of think that when a transaction is "processed" (ie message parsed and acted upon) that should be logged, but subsequently retrieving the same data should not be logged over and over.  If I could put it in simple terms I'd say think of a database context where changes are logged but reads are not.

So for me, the ideal would be to parse only once and store (+ log) that data in a format that's easy to retrieve at a later date (probably some form of database) but that would definitely be in re-architecture territory.

The issue I'm trying to solve is that every UI update or RPC call adds duplicate data to the log (because said data was already logged the first time the transaction was parsed and processed) and this can really make things "fuzzy" when reviewing logs to figure out bugs etc.

> Same here, I have no idea how this could be solved nicely. (and adding a new method just to avoid logging is already pretty over the top)

Yeah a new method isn't pretty - though you should have seen the crap I put in for the first way I handled it hehehe that was even worse!!!